python/lsst/pipe/analysis/visitAnalysis.py

```diff
@@ -593,31 +599,32 @@ def run(self, dataRefList1, dataRefList2, tract=None):
             commonZpCat1, catalog1, commonZpCat2, catalog2 = (
                 self.readCatalogs(dataRefListTract1, dataRefListTract2, "src", hscRun1=repoInfo1.hscRun,
                                   hscRun2=repoInfo2.hscRun, doReadFootprints=doReadFootprints))
-            try:
-                self.zpLabel = self.zpLabel + " " + self.catLabel
-            except:
-                pass
 
-            if repoInfo2.hscRun and self.config.doAddAperFluxHsc:
-                self.log.info("HSC run: adding aperture flux to schema...")
-                catalog2 = addApertureFluxesHSC(catalog2, prefix="")
+            # Set boolean arrays indicating sources deemed unsuitable for qa analyses
+            bad1 = self.makeBadArray(catalog1)
```

Comment:
If you get rid of the code duplication b/w `CoaddAnalysisTask` and `CompareCoaddVisitAnalysis`,  make sure this points to the same function as well.